src/test/java/StringTest.java

```diff
@@ -0,0 +1,45 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class StringTest {
+
+    @Test
+    @DisplayName("split 정상동작 테스트")
+    void split() {
+        String[] testData = "1,2".split(",");
+        assertThat(testData).contains("1","2");
```

Comment:
```suggestion
        assertThat(testData).contains("1", "2");
```
소소하지만 이런 컨벤션도 지켜주시면 좋을 것 같습니다😄